tools/rimage/src/include/rimage/sof/user/manifest.h

```diff
@@ -41,8 +41,13 @@ struct sof_man_module_type {
 	uint32_t domain_ll:1;
 	uint32_t domain_dp:1;
 	uint32_t lib_code:1;
+	uint32_t domain_rtos:1;
+	uint32_t core_type:8;
+	uint32_t user_mode:1;
+	uint32_t large_param:1;
+	uint32_t stack_on_bss:1;
+	uint32_t rsvd_:9;
 	uint32_t init_config:4; /* SOF_MAN_MOD_INIT_CONFIG_ */
```

Comment:
I'm afraid this is too late. We have already upstreamed the module type definition to Linux upstream and we cannot change this now without breaking end-user systems. FYI @abonislawski @lgirdwood @ujfalusi 

Options, 1) change non-SOF implementation, or 2) sync up in a later generation (i.e. branch the definition for a future generation).

In Linux, the old definition is used in linux/sound/soc/sof/ipc4-topology.h